jekyll/_cci2/local-cli.md

```diff
@@ -22,26 +22,28 @@ You can also run `circleci` commands in your `config.yml` file for jobs that use
 
 ## Installing
 
-### Ubuntu 16.04+, Fedora 27+, and other Distros Supporting Snap Packages
+### macOS and Linux Distros
```

Comment:
### macOS and Linux Distros -> ## Installing the CircleCI CLI on macOS and Linux Distros